anthem/lyrics/loaders.py

```diff
@@ -28,16 +31,22 @@ def load_csv(ctx, model, path, header=None, header_exclude=None, **fmtparams):
 
 
 def csv_unireader(f, encoding="utf-8", **fmtparams):
-    data = csv.reader(
-        codecs.iterencode(codecs.iterdecode(f, encoding), "utf-8"), **fmtparams
-    )
+    if sys.version_info[0] == 3:
+        data = csv.reader(codecs.iterdecode(f, encoding), **fmtparams)
```

Comment:
Are you confident about this? It seems you are getting bytes out of the file, which is not what we would expect?

Reply:
Nope, not at all :confused: